colibre/scripts/max_temperatures.py

```diff
@@ -33,16 +33,30 @@ def make_single_image(filenames, names, T_bounds, number_of_simulations, output_
     ax.set_ylabel("PDF [-]")
     ax.loglog()
 
+    log_max_Ts = []
+
     for filename, name in zip(filenames, names):
         T_max = get_data(filename)
+        log_T_max = np.log10(T_max)
         h, bin_edges = np.histogram(
-            np.log10(T_max), range=np.log10(T_bounds), bins=250, density=True
+            log_T_max, range=np.log10(T_bounds), bins=250, density=True
         )
         bins = 0.5 * (bin_edges[1:] + bin_edges[:-1])
         bins = 10 ** bins
         ax.plot(bins, h, label=name)
 
-    ax.legend()
+        log_max_Ts.append(f"{name}: {log_T_max.max():3.3f}")
```

Comment:
Perhaps, no need to have such a high precision?

```suggestion
        log_max_Ts.append(f"{name}: {log_T_max.max():3.2f}")
```